src/main/java/mate/academy/springboot/datajpa/controller/CategoryController.java

```diff
@@ -44,8 +40,9 @@ public void deleteById(@PathVariable Long id) {
         categoryService.deleteById(id);
     }
 
-    @PutMapping
-    public void update(@RequestBody CategoryResponseDto category) {
-        categoryService.update(category.getName(), category.getId());
+    @PutMapping("/{id}")
+    public void update(@RequestBody CategoryRequestDto dto,
+                       @PathVariable Long id) {
+        categoryService.save(categoryMapper.toModel(dto));

```

Comment:
Here id is ignored. Also, I think it's better to return a DTO when updating an entity.